### PR TITLE
Add cleanups to quick E2E pipeline

### PR DESCRIPTION
**Pull request by cxbrowne1207**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cmd/integration_test/build/buildspecs/quick-test-eks-a-cli.yml [hunk 1]
```diff
@@ -191,9 +196,16 @@ phases:
...
         --skip ${SKIPPED_TESTS}
         --bundles-override=${BUNDLES_OVERRIDE}
+        --cleanup-vms=true
         --test-report-folder=reports
         --branch-name=${BRANCH_NAME}
         --baremetal-branch=${BAREMETAL_BRANCH}
+  post_build:
+    commands:
+      - >
+        ./bin/test e2e cleanup vsphere
```
Comment (vignesh-goutham) [inline cmd/integration_test/build/buildspecs/quick-test-eks-a-cli.yml:206]: We'll probably need similar ones for cloudstack too

**Review by abhay-krishna — APPROVED** [review]
/approve

**Pull request merged**